### DISCUSSION TITLE
how to invoke  all handleSubmit in nested form from main submit button #react-hooks-form

### DISCUSSION THREAD

**Discussion by mustafa20010** [react-hook-form/react-hook-form#9612] [General]
i am working on this from many days but unable  to  find any solution can any one tell me about how to invoke all handleSubmit in nested form from main submit button 
this is the code sand box which provide official  in hooks form documentation 
https://codesandbox.io/s/usefieldarray-with-preview-odmtx5

**Comment by leapful** [comment]
@mustafa20010 

Could you please share what is the reason to trigger all "handleSubmit" of each nested form ?

**Comment by mustafa20010** [comment]
@leapful 
so basically i have form which i use for for single entry than i got new requirement in which i select multiple entries from table and edit them so at the end i needed all the data of forms in arr of object  from main submit button

**Comment by leapful** [comment]
@mustafa20010 

The parent form still keeps all of its nested form values. You can also just use parent form submit o get the whole data.
